conf/influxdb/influxdb.conf

```diff
@@ -25,7 +25,10 @@ reporting-disabled = true
   batch-pending = 5
   batch-timeout = "1s"
   templates = [
-    "fluo.*.*.tx.*.*.* .app.host.measurement.measurement.observer.field",
-    "fluo.*.*.*.*.* .app.host.measurement.measurement.field",
-    "fluo.*.*.*.* .app.host.measurement.measurement",
+    "fluo.class.*.*.*.*.* ..app.host.measurement.class.field",
```

Comment:
Is this still correct? It seems like you made some changes to this on the Fluo PR, but I am not sure.